Core.ApplicationServices/AdviceService.cs

```diff
@@ -246,7 +245,7 @@ private void AddRecipientByRole(Advice advice, AdviceUserRelation r, MailAddress
                     break;
                 case RelatedEntityType.itProject:
                     var projectRoles = _itProjectRights.AsQueryable().Where(I => I.ObjectId == advice.RelationId
-                        && I.Role.Name == r.Name);
+                        && I.RoleId == r.RoleId.GetValueOrDefault());
```

Comment:
when we have the foreign keys up and running then we can just load the objects from the relation in stead of this horrir